mule-migration-tool-engine/src/main/java/com/mulesoft/tools/migration/engine/project/structure/util/Exclusions.java

```diff
@@ -22,7 +22,8 @@
 
   CLASSPATH(File.separator + ".classpath"),
 
-  GIT(File.separator + ".gitignore"),
+  // GIT(File.separator + ".gitignore"),
```

Comment:
Hi, @holiday-sunrise. Thanks for this contribution. Can you revert this part (commit 0768bfe)? We can then merge the catalog bit and track the `.gitignore` part in #460.